types/stripe/index.d.ts

```diff
@@ -9845,48 +10862,145 @@ declare namespace Stripe {
-
-            createTaxId(customerId: string, data: customerTaxIds.ITaxIdCreationOptions, options: HeaderOptions, response?: IResponseFn<subscriptions.ISubscription>): Promise<customerTaxIds.ITaxId>;
-            createTaxId(customerId: string, data: customerTaxIds.ITaxIdCreationOptions, response?: IResponseFn<subscriptions.ISubscription>): Promise<customerTaxIds.ITaxId>;
-
-            listTaxIds(customerId: string, data: IListOptions, options: HeaderOptions, response?: IResponseFn<IList<sources.ISource>>): IListPromise<sources.ISource>;
-            listTaxIds(customerId: string, data: IListOptions, response?: IResponseFn<IList<sources.ISource>>): IListPromise<sources.ISource>;
-            listTaxIds(customerId: string, options: HeaderOptions, response?: IResponseFn<IList<subscriptions.ISubscription>>): IListPromise<subscriptions.ISubscription>;
-            listTaxIds(customerId: string, response?: IResponseFn<IList<subscriptions.ISubscription>>): IListPromise<subscriptions.ISubscription>;
-
-            retrieveTaxId(customerId: string, taxId: string, options: HeaderOptions, response?: IResponseFn<IStripeSource>): Promise<IStripeSource>;
-            retrieveTaxId(customerId: string, taxId: string, response?: IResponseFn<IStripeSource>): Promise<IStripeSource>;
-
-            deleteTaxId(customerId: string, taxId: string, options: HeaderOptions, response?: IResponseFn<IDeleteConfirmation>): Promise<IDeleteConfirmation>;
-            deleteTaxId(customerId: string, taxId: string, response?: IResponseFn<IDeleteConfirmation>): Promise<IDeleteConfirmation>;
+            deleteSubscriptionDiscount(
+                customerId: string,
+                subscriptionId: string,
+                options: HeaderOptions,
+                response?: IResponseFn<IDeleteConfirmation>,
+            ): Promise<IDeleteConfirmation>;
+            deleteSubscriptionDiscount(
+                customerId: string,
+                subscriptionId: string,
+                response?: IResponseFn<IDeleteConfirmation>,
+            ): Promise<IDeleteConfirmation>;
+
+            createTaxId(
+                customerId: string,
+                data: customerTaxIds.ITaxIdCreationOptions,
+                options: HeaderOptions,
+                response?: IResponseFn<subscriptions.ISubscription>,
+            ): Promise<customerTaxIds.ITaxId>;
+            createTaxId(
+                customerId: string,
+                data: customerTaxIds.ITaxIdCreationOptions,
+                response?: IResponseFn<subscriptions.ISubscription>,
+            ): Promise<customerTaxIds.ITaxId>;
+
+            listTaxIds(
+                customerId: string,
+                data: IListOptions,
+                options: HeaderOptions,
+                response?: IResponseFn<IList<sources.ISource>>,
+            ): IListPromise<sources.ISource>;
```

Comment:
Are the return types for listTaxIds correct? Maybe better return ```customerTaxIds.ITaxId```?

Reply:
I think you might be right @Jank1310 , but it is not related to the topic of the PR. 
I added support for tax rates, customer tax id's are a different thing. 
But I ran prettier so it looks like I changed those lines as well, sorry for that.